### PR TITLE
Update link_aliasing.md

### PR DESCRIPTION
**Pull request by gsinghAppboy**
# Pull Request/Issue Resolution

**Description of Change:**
Adding in more documentation about link segmentation and action based filters

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_hidden/private_betas/link_aliasing.md [hunk 1]
```diff
@@ -72,10 +72,37 @@ For new message variants, any existing link template can be used from the __Link
...
 
+
 ## Link Segmentation
-Two new segmentation filters are now available, allowing you to create segmentation filters based on clicking a specific tracked alias. The filter only displays campaigns or Canvases that have tracked aliases present.
+
+Retargeting of aliases is now available as a segmentation filter.  The two new filters allow you to create segmentation filters based on your customers clicking a specific tracked alias, from either an email campaign or canvas step. The filter only displays Campaigns or Canvases that have tracked aliases present.
+
+### Tracking a Link
+
+When composing your email message, a new column will be present in the "Link Management" tab.  You can now indicate to Braze which alias you would like to be 'tracked' for segmentation purposes.  Only aliases that you have indicated to be tracked will be present in segmentation filters.
```
Comment (willng-braze) [inline _docs/_hidden/private_betas/link_aliasing.md:82]: Do we need to indicate that there is a max of 100 across all appgroups.  And that links can be returned to the pool by untracking links or archiving campaigns.

Comment (gsinghAppboy) [reply _docs/_hidden/private_betas/link_aliasing.md:82]: it's documented below this in the 'link tracking' section
https://github.com/braze-inc/braze-docs/blob/0ebeff5cffa8dd5f22059efefa4866a8f3ba7519/_docs/_hidden/private_betas/link_aliasing.md#link-tracking

File: _docs/_hidden/private_betas/link_aliasing.md [hunk 2]
```diff
@@ -72,10 +72,37 @@ For new message variants, any existing link template can be used from the __Link
...
 ## Link Segmentation
-Two new segmentation filters are now available, allowing you to create segmentation filters based on clicking a specific tracked alias. The filter only displays campaigns or Canvases that have tracked aliases present.
+
+Retargeting of aliases is now available as a segmentation filter.  The two new filters allow you to create segmentation filters based on your customers clicking a specific tracked alias, from either an email campaign or canvas step. The filter only displays Campaigns or Canvases that have tracked aliases present.
+
+### Tracking a Link
+
+When composing your email message, a new column will be present in the "Link Management" tab.  You can now indicate to Braze which alias you would like to be 'tracked' for segmentation purposes.  Only aliases that you have indicated to be tracked will be present in segmentation filters. 
+
+_Note: Tracked Aliases are only for segmentation purposes and have no impact on your link being 'tracked' for reporting purposes._
```
Comment (willng-braze) [inline _docs/_hidden/private_betas/link_aliasing.md:84]: Probably worth mentioning what the implications of are untracking a link is.

Untracking a link will not deallocate existing segments with the filter to the untracked alias.  The old data will remain on the user profiles until they are evicted by newer data.  The filter will continue to exist but new segments cannot be created with that filter.

**Review by willng-braze — APPROVED** [review]

File: _docs/_hidden/private_betas/link_aliasing.md (hunk 2 above)
Comment (KellieHawks) [reply _docs/_hidden/private_betas/link_aliasing.md]: Updated and addressed. Thanks!

**Review by KellieHawks — APPROVED** [review]
Looks good, made some edits from feedback

**Pull request merged**